### PR TITLE
29 migrate to version 3

### PR DESCRIPTION
**Pull request by lpandath**
## Description
Version 3 of DAMAP

What does this PR do?

Supports:
HE template support
University project search
University person search
Strings and translations

### Checks

- [ ] Summary updated
- [ ] Version view updated
- [ ] Documentation added
- [ ] Tests added
- [ ] E2e tests created
- [ ] Successfully ran e2e tests before merge

closes GH-

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -33,13 +28,13 @@ jobs:
...
         with:
-          report_paths: 'reports/lint/**/*.xml'
+          report_paths: "**/reports/lint/**/*.xml"
 
       - name: Publish Test Report
-        uses: mikepenz/action-junit-report@v3
+        uses: mikepenz/action-junit-report@v4
         if: always()
         with:
           report_paths: 'reports/test/**/*.xml'
```
Comment (rekt-hard) [inline .github/workflows/test.yml:40]:
```suggestion
          report_paths: "**/reports/unit/**/*.xml"
```

File: Dockerfile [hunk 2]
```diff
@@ -2,18 +2,18 @@
...
-        org.label-schema.schema-version="1.0" \
-        org.label-schema.docker.cmd="docker run -d -p 8080:8080 damap"
+    org.label-schema.name="DAMAP-frontend" \
+    org.label-schema.description="DAMAP is a tool that aims to facilitate the creation of data management plans (DMPs) for researchers." \
+    org.label-schema.usage="https://github.com/tuwien-csd/damap-frontend/tree/master/README.md" \
+    org.label-schema.vendor="Technische Universität Wien" \
+    org.label-schema.url="https://github.com/tuwien-csd/damap-frontend" \
+    org.label-schema.vcs-url="https://github.com/tuwien-csd/damap-frontend" \
+    org.label-schema.schema-version="1.0" \
+    org.label-schema.docker.cmd="docker run -d -p 8080:8080 damap"
```
Comment (rekt-hard) [inline Dockerfile:16]: minor: we can update these with TU Graz specific values
Feel free to just create an issue to tackle later

File: apps/damap-frontend/src/app/components/layout/layout.component.html [hunk 3]
```diff
@@ -63,17 +97,27 @@ <h1 translate>title</h1>
...
   </div>
   <div class="info">
-    <a [href]="'layout.impressum.link' | translate" rel="noopener" target="_blank">{{'layout.impressum.text' |
-      translate}}</a>
-    <a [href]="'layout.termsandconditions.link' | translate" rel="noopener"
-      target="_blank">{{'layout.termsandconditions.text' | translate}}</a>
-    <a [href]="'layout.dataprotection.link' | translate" rel="noopener" target="_blank">{{'layout.dataprotection.text' |
-      translate}}</a>
-    <a [href]="'layout.accessibility.link' | translate" rel="noopener" target="_blank">{{'layout.accessibility.text' |
-      translate}}</a>
```
Comment (rekt-hard) [inline apps/damap-frontend/src/app/components/layout/layout.component.html:77]: accessibility declaration should remain for TU Graz requirements

**Review by rekt-hard — COMMENTED** [review]
Please also include the changes from this PR regarding google API calls: https://github.com/tuwien-csd/damap-frontend/pull/181/files#diff-34a6ada45680e589b93700254a24eef7e0c4844d5f4b5993b4568d3cfa5e991aL8-L13

File: .github/workflows/release.yml [hunk 4]
```diff
@@ -21,18 +21,18 @@ jobs:
     steps:
      # sed -i '/# create a second/,$d' Dockerfile
       - name: Checkout
-        uses: actions/checkout@v3
+      - uses: actions/checkout@v4
```
[the remark this answers is not in the document]
Comment (lpandath) [reply .github/workflows/release.yml:24]: @rekt-hard whats the suggestion here, im not that sure: if '-' in the core is the same?

[the remark this answers is not in the document]
Comment (rekt-hard) [reply .github/workflows/release.yml:24]: The `-` in .yaml files denotes an entry/object. Our entry already starts with `- name: Checkout`. Hence, we do not need the `-` prior to `uses`

**Review by rekt-hard — APPROVED** [review]
The tooltips starting with `help` are not resolved properly in our instance:
image

Since we have an email here instead of a link, prefixing it with `mailto:` will open the e-mail client. These should be addressed at all occurrences:
image

Will create separate issues for these.

Other than that, looks good :)

**Pull request merged**